man/man8/zfs.8

```diff
@@ -268,6 +268,21 @@ zfs \- configures ZFS file systems
 .LP
 .nf
 \fBzfs\fR \fBdiff\fR [\fB-FHt\fR] \fIsnapshot\fR \fIsnapshot|filesystem\fR
+.fi
+
+.LP
+.nf
+\fB\fBzfs key -l\fR \fIfilesystem\fR | \fIvolume\fR\fR
```

Comment:
I can change this if you'd like (its a bit tedius to make every one of these a separate funnction in both kernel and userspace. My only other qualm here is that people have asked for a few other key-related subcommands like `zfs verifykey` which would tell you if a key is correct (even if its already loaded).

Another one I was going to add while doing all these fixups is `zfs linkkey` which would tell a dataset to inherit the wrapping keys of a parent. Currently, you can break wrapping key inheritance with `zfs key -c` but there's no way to relink them later so this would add that functionality. So at that point there are now 5 key commands which seems like a lot of bloat in both `zfs_cmd.c` and `zfs_ioctl.c`.

I'm fine doing it either way. Let me know what you think is best. Maybe @behlendorf could weigh in here too.